Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -10956,6 +10957,19 @@ - (void)imageAnalysisGestureDidBegin:(WKImageAnalysisGestureRecognizer *)gesture
     } forRequest:request];
 }
 
+#if ENABLE(IMAGE_ANALYSIS_FOR_MACHINE_READABLE_CODES)
+static BOOL shouldUseMachineReadableCodeMenuFromImageAnalysisResult(CocoaImageAnalysis *result)
+{
+#if HAVE(BCS_LIVE_CAMERA_ONLY_ACTION_SPI)
+    return [result.barcodeActions indexOfObjectPassingTest:^BOOL(BCSAction *action, NSUInteger index, BOOL *stop) {
+        return [action respondsToSelector:@selector(isLiveCameraOnlyAction)] && [(id)action isLiveCameraOnlyAction];
+    }] == NSNotFound;
```

Comment:
IIUC, the live camera action should be the only action in the MRC menu in this case? (I wonder if it makes sense to filter out these live camera only actions instead, so we still add any other remaining items. If, in practice, the live camera action appears alone, then this is probably fine).

Reply:
Hm...I see. This is probably okay, then, if this scanned action only appears by itself in the `mrcMenu`.

Just to clarify, are the QR codes in https://whsieh.github.io/examples/qr-code considered to be "scanned actions"?

Reply:
I spoke with @whsieh in Slack, and included some more background in <rdar://98207615>.